src/bundle/Resources/views/admin/language/view.html.twig

```diff
@@ -23,53 +23,65 @@
 
 {% block content %}
     <section class="container mt-4 px-5">
-        <div>
-            <dl>
-                <dt>{{ 'language.name.label'|trans|desc('Name') }}</dt>
-                <dd>{{ language.name }}</dd>
+        <header class="ez-table-header round">
+            <h5>{{ 'language.information.header'|trans|desc('Language information') }}</h5>
 
-                <dt>{{ 'language.code.label'|trans|desc('Language code') }}</dt>
-                <dd>{{ language.languageCode }}</dd>
+            <div>
+                {{ form_start(deleteForm, {"action": path("ezplatform.language.delete", {"languageId": language.id, "redirectErrorsTo": "view"})}) }}
 
-                <dt>{{ 'language.id.label'|trans|desc('ID') }}</dt>
-                <dd>{{ language.id }}</dd>
+                {{ form_widget(deleteForm.language) }}
+                <button type="button"
+                        class="btn btn-danger"
+                        data-toggle="modal"
+                        data-target="#delete-language-modal">
+                    <svg class="ez-icon">
+                        <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#trash"></use>
+                    </svg>
+                    {{ 'language.form.delete'|trans|desc('Delete') }}
```

Comment:
In order to have a better consistency with all the tables, could you please remove the `Delete`. Just keep the icon.
Thanks